Makefile

```diff
@@ -15,15 +15,11 @@ lint:
 
 .PHONY: test
 test:
-	pytest --cov=aiohttp_devtools --boxed --duration 5 && (coverage combine || test 0)
-
-.PHONY: testcov
-testfast:
-	pytest --cov=aiohttp_devtools --boxed --fast -n 4 && (echo "building coverage html"; coverage combine; coverage html)
+	pytest --cov=aiohttp_devtools --forked --duration 5 && (coverage combine || test 0)
```

Comment:
Why duplicate this in every script when you can just put it into the config?

Reply:
I don't know. I'm happy to just remove the whole thing. I'm not using it and it doesn't appear to be used anywhere else.

Reply:
Note that `--forked` / `--boxed` essentially isolates every test but the marker decorators `@pytest.mark.forked` only do that to specific tests: https://github.com/pytest-dev/pytest-forked#usage-examples.

It looks like the CLI arg is just unnecessary.